kit/ChildSession.hpp

```diff
@@ -90,6 +90,13 @@ class ChildSession final : public Session
         return _docManager->sendFrame(msg.data(), msg.size(), WSOpCode::Text);
     }
 
+    bool sendSaveFrame(const std::string& msg)
+    {
+        bool ret = sendTextFrame(msg);
+        _docManager->handleSaveMessage(msg);
```

Comment:
Isn't it simpler to just do this line above, at line 3007, if `saveCommand` is true, after we `sendTextFrame`? Unless we expect this to be called from multiple places, or to become more complex, that is.

Reply:
Sure - why not; fixed.